sympy/functions/elementary/complexes.py

```diff
@@ -467,7 +466,8 @@ def eval(cls, arg):
                     return Abs(base)**exponent
                 if base.is_positive == True:
                     return base**re(exponent)
-                return (-base)**re(exponent)*exp(-S.Pi*im(exponent))
+                if base.is_negative:
+                    return (-base)**re(exponent)*exp(-S.Pi*im(exponent))
```

Comment:
@jksuom Travis points build to fail w.r.t to the inequalities file, but I din't change that file?